usefulUtilities/spawnPointZone/entityResources/entityClientScripts/spawnPointZone.js

```diff
@@ -11,8 +11,10 @@
 (function () {
     var _this;
 
-    var DEBUG = 0;
+    var DEBUG = 1;
```

Comment:
do you want debug on in the production version?

Reply:
nope